src/styles/LoginScreen/Components_Register.jsx

```diff
@@ -0,0 +1,102 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '../colors';
+
+export const containers = StyleSheet.create({
+  containerRegisterForm: {
+    alignContent: 'center',
+    alignSelf: 'center',
+    flexDirection: 'column',
+  },
+});
+
+export const formRegister = StyleSheet.create({
+  inputUserName: {
+    fontSize: 20,
+    color: COLORS.GRAY,
+    marginTop: 10,
+    marginBottom: 5,
+    fontWeight: 'bold',
+    fontFamily: 'Nunito_500Medium',
+    borderColor: COLORS.WHITE,
+    backgroundColor: COLORS.WHITE,
+    borderWidth: 1,
+    width: 330,
+    height: 45,
+    padding: 10,
+    fontSize: 13,
+    fontWeight: '300',
+    marginBottom: 10,
+    borderRadius: 8,
+    bottom: 15,
+  },
+  inputEmail: {
+    fontSize: 20,
+    color: COLORS.GRAY,
+    marginTop: 10,
+    marginBottom: 5,
+    fontWeight: 'bold',
+    fontFamily: 'Nunito_500Medium',
+    borderColor: COLORS.WHITE,
+    backgroundColor: COLORS.WHITE,
+    borderWidth: 1,
+    width: 330,
+    height: 45,
+    padding: 10,
+    fontSize: 13,
+    fontWeight: '300',
+    marginBottom: 10,
+    borderRadius: 8,
+  },
+  inputPassword: {
+    fontSize: 20,
+    color: COLORS.GRAY,
+    marginTop: 10,
+    marginBottom: 5,
+    fontWeight: 'bold',
+    fontFamily: 'Nunito_500Medium',
+    borderColor: COLORS.WHITE,
+    backgroundColor: COLORS.WHITE,
+    borderWidth: 1,
+    width: 330,
+    height: 45,
+    padding: 10,
+    fontSize: 13,
+    fontWeight: '300',
+    marginBottom: 10,
+    borderRadius: 8,
+    top: 15,
+  },
```

Comment:
Could you please refactor this? I see you have the same props but you're just changing the bottom and top styles